src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php

```diff
@@ -44,6 +44,7 @@ protected function tableBuildSettings(array $records) {
     return [
       'data' => $this->getTableRows($records),
       'columns' => $this->getTableHeader(),
+      'tableOptions' => $this->parseTableOptions(),
```

Comment:
Can we please remove this line from here and add the following to `Table.php` below `Table::rowHeaderField`?

```
/**
 * {@inheritdoc}
 */
protected function tableBuildSettings(array $records) {
  $settings = parent::tableBuildSettings($records);

  $settings['tableOptions']['pageLength'] = $this->config('table', 'options', 'page_length');
  $settings['tableOptions']['searching'] = $this->config('table', 'options', 'searching');

  return $settings;
}
```